styleguide/source/_meta/_00-head.twig

```diff
@@ -7,6 +7,7 @@
 
     <link rel="stylesheet" href="https://use.typekit.net/eie0akm.css" />
     <link rel="stylesheet" href="../../assets/css/reset.css?{{ cacheBuster }}" />
+    <link rel="stylesheet" href="../../assets/css/panels-ipe.css?{{ cacheBuster }}" media="all" />
```

Comment:
Why wouldn't this be packaged up with the rest of `style.css`?

Reply:
Everything in `panels-ipe.css` should already be included in Drupal. It's the styling from the IPE module that I copied and pasted into its own file in the style guide. I needed to display it in the style guide for parity reasons, but didn't actually want to package it up with everything else because it doesn't need to be included in the dev assets. Is there a better way to go about this?

Reply:
Got it. Sorry, I didn't understand that. Just put a comment here as discussed and we'll be set.